### DISCUSSION TITLE
关于区块链 - 我们能用它干点啥

### DISCUSSION THREAD

**Discussion by guttasingapore** [RoadSideUniversity/ClubHouse2024#14] [Ideas]
BLOCKCHAIN FESTIVAL SINGAPORE 2024
https://blockchainfestival.asia/
#BFSG #BFSG24 #BFSG2024
我永远都在不得要领的学习中

**Comment by Lewisrxliu** [comment]
https://www.fintechfestival.sg
下周有这样一个高逼格的展会 新加坡金融科技大会，各位同学们，有啥想法都请分享一下吧，鄙人可以代表大家观察观察 试探试探 
15 - 17 November 2023 
好点子永远是稀缺的资源

**Reply by guttasingapore** [reply]
2025 SFF FinTech Festival 两年后的今天, 不胜唏嘘！看上去冬天已经来了，这个冬天会有多长我们不知道，来年春天才能知道会冻死多少人!

https://www.fintechfestival.sg

![image](https://github.com/user-attachments/assets/67ac58f5-5458-49dd-8436-735e2f4ee144)

![image](https://github.com/user-attachments/assets/0903ea23-527e-48db-b376-2585032f6fa3)

![image](https://github.com/user-attachments/assets/f749cc75-42b4-42c8-84d4-17bdca1a976c)

![image](https://github.com/user-attachments/assets/15ec3dbb-cb0b-43f3-a870-a78ddd9f647a)

**Reply by guttasingapore** [reply]
原来他们嗨的是这个
https://www.zaobao.com.sg/realtime/singapore/story20251112-7807639#google_vignette
PayLah!、GrabPay和ShopeePay用户下来可在中国微信扫码支付
2025年11月13日

本地消费者接下来可通过PayLah!、GrabPay和ShopeePay等电子钱包，在中国直接扫微信QR码支付，或展示钱包内QR码进行支付，在中国旅游时有更多可行的支付选项。

另外，中国银行新加坡分行的手机应用，下来也能在中国扫微信QR码支付。

腾讯旗下跨境支付平台TenPay Global星期三（11月12日）宣布，已与多方达成战略合作，将在未来分阶段完成。

其他的本地合作方包括电子钱包Starryblu和EvoNet支付网关。

这些合作是腾讯旗下TenPay Global进一步扩张海外支付合作伙伴网络的一部分。


TenPay Global说，合作的海外钱包用户不需要下载微信或进行额外设置，就能直接在中国内地的数千万商家内进行扫码支付。


星展银行新加坡消费银行支付与平台业务部主管陈秀娴说，双方合作重点在于提升支付互联互通，让跨境交易的来往运作更便捷，也更易于规模扩张。

“例如，让我们的300万PayLah!用户能够直接在中国扫微信QR码进行支付。”

此外，星展也在通过腾讯的微信生态探索更多创新使用案例；例如，整合集成微信小程序，或让用户能在星展的平台直接在线支付。接下来，双方合作还将分阶段实现商户展示QR码交易、或消费者展示QR码支付、线上支付等功能。

中国银行新加坡分行与TenPay Global星期四（13日）签署谅解备忘录，除了手机银行扫码，中国银行新加坡分行银行卡也可安全绑定至微信支付。

中国银行新加坡分行行长史炜表示，此次与TenPay Global的携手，不仅是落实两国数码经济领域合作的重要举措，也有助于提升新加坡赴华人士的支付体验，促进中新人文交流。

根据备忘录框架，双方未来将继续深化合作，中长期目标是在确保合规与安全的前提下，持续优化跨境支付体验，探索在海外华人华侨汇款、跨境结算及支付创新等领域的更多应用，共同构建更包容高效的跨境金融服务生态

**Comment by Lewisrxliu** [comment]
https://www.blocktempo.com/binance-founder-changpeng-zhao-agrees-to-step-down-plead-guilty/

**Comment by Lewisrxliu** [comment]
> https://www.blocktempo.com/binance-founder-changpeng-zhao-agrees-to-step-down-plead-guilty/

U.S. DOJ Reveals That Prosecutors May Argue That CZ Should Be Sentenced To 10 Years In Prison

**Comment by guttasingapore** [comment]
> > https://www.blocktempo.com/binance-founder-changpeng-zhao-agrees-to-step-down-plead-guilty/
> 
> U.S. DOJ Reveals That Prosecutors May Argue That CZ Should Be Sentenced To 10 Years In Prison

[DOJReply.pdf](https://github.com/RoadSideUniversity/ClubHouse2023Nov/files/13483651/DOJReply.pdf)

**Comment by Lewisrxliu** [comment]
关于钱币，票据与彩票 Dec-2023 游记，部分解释了鄙人为什么没有购买NFT的欲望
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/678d06a3-ffa7-491c-8f3a-ba4a618334e8)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/33abf192-4e27-4e7a-b2fe-079341f4d94d)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/82ab08a5-8d69-47ff-876f-9efced1308a7)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/21fbf3f0-5281-410d-ae49-0814db3d6a4e)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/fe74b8cd-de9f-4154-b2d9-f8b2a7ca0dcc)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/4437d130-2a53-46b3-91ba-4c548178e8aa)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/2a947f12-026d-4634-877d-f7b68d6796cc)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/190e1e4b-f434-4961-be5d-977a790967a0)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/d811c535-8dbb-4465-b31c-c90f53404c8e)
